src/misc2.c

```diff
@@ -921,186 +921,232 @@ static char_u modifier_keys_table[] =
+
+    {TRUE, K_F4, STRING_INIT("F4"), NULL},
+    {TRUE, K_F5, STRING_INIT("F5"), NULL},
+    {TRUE, K_F6, STRING_INIT("F6"), NULL},
+    {TRUE, K_F7, STRING_INIT("F7"), NULL},
+    {TRUE, K_F8, STRING_INIT("F8"), NULL},
+    {TRUE, K_F9, STRING_INIT("F9"), NULL},
+
+    {TRUE, K_FOCUSGAINED, STRING_INIT("FocusGained"), NULL},
+    {TRUE, K_FOCUSLOST, STRING_INIT("FocusLost"), NULL},
+    {TRUE, K_HELP, STRING_INIT("Help"), NULL},
+    {TRUE, K_HOME, STRING_INIT("Home"), NULL},
+    {TRUE, K_IGNORE, STRING_INIT("Ignore"), NULL},
+    {TRUE, K_INS, STRING_INIT("Ins"),
+	&key_names_table[IDX_KEYNAME_INS].name},		// Alternative name
+    {TRUE, K_INS, STRING_INIT("Insert"), NULL},
+    {
 #ifdef FEAT_MOUSE_JSB
-    {K_JSBTERM_MOUSE,	(char_u *)"JsbMouse"},
+    TRUE,
+#else
+    FALSE,
 #endif
-#ifdef FEAT_MOUSE_PTERM
-    {K_PTERM_MOUSE,	(char_u *)"PtermMouse"},
+	K_JSBTERM_MOUSE, STRING_INIT("JsbMouse"), NULL},
+    {TRUE, K_K0, STRING_INIT("k0"), NULL},
+    {TRUE, K_K1, STRING_INIT("k1"), NULL},
+    {TRUE, K_K2, STRING_INIT("k2"), NULL},
+    {TRUE, K_K3, STRING_INIT("k3"), NULL},
+    {TRUE, K_K4, STRING_INIT("k4"), NULL},
+    {TRUE, K_K5, STRING_INIT("k5"), NULL},
+    {TRUE, K_K6, STRING_INIT("k6"), NULL},
+    {TRUE, K_K7, STRING_INIT("k7"), NULL},
+    {TRUE, K_K8, STRING_INIT("k8"), NULL},
+    {TRUE, K_K9, STRING_INIT("k9"), NULL},
+
+    {TRUE, K_KDEL, STRING_INIT("kDel"), NULL},
+    {TRUE, K_KDIVIDE, STRING_INIT("kDivide"), NULL},
+    {TRUE, K_KEND, STRING_INIT("kEnd"), NULL},
+    {TRUE, K_KENTER, STRING_INIT("kEnter"), NULL},
+    {TRUE, K_KHOME, STRING_INIT("kHome"), NULL},
+    {TRUE, K_KINS, STRING_INIT("kInsert"), NULL},
+    {TRUE, K_KMINUS, STRING_INIT("kMinus"), NULL},
+    {TRUE, K_KMULTIPLY, STRING_INIT("kMultiply"), NULL},
+    {TRUE, K_KPAGEDOWN, STRING_INIT("kPageDown"), NULL},
+    {TRUE, K_KPAGEUP, STRING_INIT("kPageUp"), NULL},
+    {TRUE, K_KPLUS, STRING_INIT("kPlus"), NULL},
+    {TRUE, K_KPOINT, STRING_INIT("kPoint"), NULL},
+    {TRUE, K_LEFT, STRING_INIT("Left"), NULL},
+    {TRUE, K_LEFTDRAG, STRING_INIT("LeftDrag"), NULL},
+    {TRUE, K_LEFTMOUSE, STRING_INIT("LeftMouse"), NULL},
+    {TRUE, K_LEFTMOUSE_NM, STRING_INIT("LeftMouseNM"), NULL},
+    {TRUE, K_LEFTRELEASE, STRING_INIT("LeftRelease"), NULL},
+    {TRUE, K_LEFTRELEASE_NM, STRING_INIT("LeftReleaseNM"), NULL},
+    {TRUE, NL, STRING_INIT("LF"),
+	&key_names_table[IDX_KEYNAME_NL].name},			    // Alternative name
+    {TRUE, NL, STRING_INIT("LineFeed"),
+	&key_names_table[IDX_KEYNAME_NL].name},			    // Alternative name
+    {TRUE, '<', STRING_INIT("lt"), NULL},
+    {TRUE, K_MIDDLEDRAG, STRING_INIT("MiddleDrag"), NULL},
+    {TRUE, K_MIDDLEMOUSE, STRING_INIT("MiddleMouse"), NULL},
+    {TRUE, K_MIDDLERELEASE, STRING_INIT("MiddleRelease"), NULL},
+    {TRUE, K_MOUSE, STRING_INIT("Mouse"), NULL},
+    {TRUE, K_MOUSEDOWN, STRING_INIT("MouseDown"),
+	&key_names_table[IDX_KEYNAME_SWU].name},		    // OBSOLETE: Use ScrollWheelXXX instead
```

Comment:
Since `MouseDown` and `MouseUp` no longer neighbor each other, I think it's better to use two different comments:
```suggestion
	&key_names_table[IDX_KEYNAME_SWU].name},		    // OBSOLETE: Use ScrollWheelDown instead
```

Reply:
Got call.